section3/reflection.md

```diff
@@ -2,12 +2,24 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+* I'm still nervous about the amount of time everything takes me, but I think it's somewhat fun. I've gone down a few rabbit holes, not any where I felt I wasn't making progress, just finding really interesting articles and documentation on code. Even, with more support removed, it's starting to feel like I'm solving little mysteries as opposed to daunting problems.
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
+* I'm a daily Spotify listener. I think Spotify uses arrays in most of it's app. Play lists are comprised of songs. Variables named song lists, probably have songs as elements in arrays. Perhaps, it's something like `var songs = ["About Damn Time", "Crybaby", "Sidelines", "Bad Guy"]`.
+
 3. In your own words, explain what an Array method is.
 
+* An Array method is a function that is already built into JavaScript. It allows us to manipulate an array to do something we need it to do, like add an element to the inside of an array or shorten it.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
 
+* A loop allows us to perform the same function or several functions over and over again, so we don't have to manually write the function a million times, which is impossible. We use a loop in conjunction with an array to perform a calculation or make each individual element in an array do something. The loop iterates or repeats the same step over each part of the array. Arrays aren't really needed if we are using loops to perform calculations, for example when setting up a clock or a counter of some sort.
+
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+* Think of a loop like a bicycle wheel on the front of a bike. The wheel has a star stamp on it that let's you leave a cool paint pattern on the ground. When you ride your bike the stamp picks up paint at the top and as the wheel comes down it leaves a star on the ground. You can leave as many stars on the ground as you want, as long as you set yourself up with enough paint.
+
 6. What questions do you still have about Arrays and loops?
+
+* I'm try to figure out if and how it's possible to combine arrays with logical and comparison operators in conditional statement. I can't quite visualize how I would create a function that iterated through an array while comparing the logic in a conditional statement.
```

Comment:
Imagine you work at Instagram, and you are displaying the pictures on someone's feed. You could have an array of all the pictures of user's that they follow, but if they chose the option to not see someone's posts for a while, you have a conditional statement that would chose to not display the pictures from that user. These concepts usually start to clear up once you are working on larger projects.